Orange/widgets/utils/__init__.py

```diff
@@ -23,7 +24,7 @@ def vartype(var):
 
 
 def progress_bar_milestones(count, iterations=100):
-    return set([int(i*count/float(iterations)) for i in range(iterations)])
+    return set(int(i*count/float(iterations)) for i in range(iterations))
```

Comment:
I agree that this is certainly an improvement, but
`{int(i * count / float(iterations)) for i in range(iterations)}` could be even better. :)

Reply:
Come on :) I just did this for the lint's sake.

Reply:
I know it was for lint. I just wanted to improve the improvement.

Same below. And below. :)